<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Output</class>
 <widget class="QDialog" name="Output">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decklink Output</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="propertiesLayout"/>
  </widget>
  <widget class="QPushButton" name="startOutput">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopOutput">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Keyer output requires RGB mode in advanced settings.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
